pages/docs.html

```diff
@@ -499,6 +499,61 @@ <h5>SIGNUP</h5>
 
     <!----------form-end---------->
 
+
+    <!----------toast-start---------->
+
+    <div class='toast toast-danger'>
+        <div class='msg'>Can't send photo. Retry in 5 seconds.</div>
+        <button class='icon-only-button toast-button font-semibold'>RETRY</button>
+        <button class='icon-only-button toast-button'><img src='/assets/close-icon.svg'></button>
+    </div>
+
+    <div class='toast toast-success'>
+        <div class='msg'>Downloaded</div>
+        <button class='icon-only-button toast-button'><img src='/assets/close-icon.svg'></button>
+    </div>
+    
+    <div class='toast toast-info'>
+        <div class='msg'>Downloading</div>
+        <button class='icon-only-button toast-button'><img src='/assets/close-icon.svg'></button>
+    </div>
+
+    <!----------toast-end---------->
+    
+
+    <!-----simplified-grid-start----->
+
+    <div class="grid-container">
+        <div class="grid-50-50">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+        </div>
+    </div>
+
+    <div class="grid-container">
+        <div class="grid-30-70">
+            <img class='res-img center-content' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+        </div>
+    </div>
+
+    <div class="grid-container">
+        <div class="grid-70-30">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img center-content' src="/assets/res-img-demo.jpg" alt="image">
+        </div>
+    </div>
+
+    <div class="grid-container">
+        <div class="grid-3-column">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+            <img class='res-img' src="/assets/res-img-demo.jpg" alt="image">
+        </div>
+    </div>
+
+    <!-----simplified-grid-end----->
```

Comment:
clean and understandable. Good that you have placed comments to segregate sections

Reply:
Thanks for your feedback.